jpa/subsystem/src/main/java/org/jboss/as/jpa/processor/JPAClassFileTransformerProcessor.java

```diff
@@ -41,6 +44,13 @@
  */
 public class JPAClassFileTransformerProcessor implements DeploymentUnitProcessor {
 
+    private static final boolean hibernate51CompatibilityTransformer = Boolean.getBoolean("Hibernate51CompatibilityTransformer");
```

Comment:
This should probably use `Boolean.parseBoolean(WildFlySecurityManager.getPropertyPrivileged("Hibernate51CompatibilityTransformer", "false))`.

Reply:
fixing.

Reply:
fixing.